stdlib/2/subprocess.pyi

```diff
@@ -105,8 +105,6 @@ class Popen(Generic[_T]):
     def send_signal(self, signal: int) -> None: ...
     def terminate(self) -> None: ...
     def kill(self) -> None: ...
-    def __enter__(self) -> Popen: ...
-    def __exit__(self, type, value, traceback) -> None: ...
```

Comment:
These are neither described in the documentation, nor could I find them in the implementation, and a short test also showed they don't exist at runtime.